x-pack/plugins/actions/server/plugin.test.ts

```diff
@@ -50,25 +50,14 @@ describe('Actions Plugin', () => {
       };
     });
 
-    it('should log warning when Encrypted Saved Objects plugin is using an ephemeral encryption key', async () => {
-      // coreMock.createSetup doesn't support Plugin generics
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      await plugin.setup(coreSetup as any, pluginsSetup);
-      expect(pluginsSetup.encryptedSavedObjects.usingEphemeralEncryptionKey).toEqual(true);
-      expect(context.logger.get().warn).toHaveBeenCalledWith(
-        'APIs are disabled because the Encrypted Saved Objects plugin uses an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or use the bin/kibana-encryption-keys command.'
-      );
-    });
-
     describe('routeHandlerContext.getActionsClient()', () => {
-      it('should not throw error when ESO plugin not using a generated key', async () => {
```

Comment:
Now that `actions` is dependent on `ESO` I think we can remove this all together, no?
It should never be possible for `getActionsClient` to be available when ESO is not... right? 🤔